src/kessler.F

```diff
@@ -1015,36 +1015,25 @@ subroutine geterain_GPU(dt,cpx,lx1,erain,ruh,rvh,t,rr,q3d,vr)
       double precision :: budt
```

Comment:
I thought that we were not going to remove any B4B regions in this PR.

Reply:
Actually I am not sure why we still want to keep the B4B regions in the CM1 code since we are not using them at all. Thus I prefer to clean them up while I am changing the code style.

Reply:
OK, I am fine with that.